src/Components/home/Home.tsx

```diff
@@ -1,31 +1,41 @@
 import React, {useState, useEffect} from "react";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import axios from 'axios'
 
-export function Home(props) {
-    const [employees, setEmployees] = useState([{}])
-    
+type Employee = {
+    id: string,
+    firstName: string,
+    lastName: string,
+    phone: string,
+}
+
+export function Home(): React.JSX.Element {
+    const [employees, setEmployees] = useState<Employee[]>([])
+    const navigate = useNavigate();
+
     const URI = 'https://localhost:7189/api/employees'
-    let counter = 1;
+    let counter: number = 1;
 
-    function handleDeleteButtonClick(id) {
+    function handleDeleteButtonClick(id: string) {
         axios.delete(URI + `/${id}`)
         .then((response) => { 
-            document.location = 'http://localhost:3000'
             alert('Successfully deleted employee!');
+            navigate("/");
         })
         .catch((error) => console.log(error));
     }
 
     useEffect(() => {
         axios.get(URI)
             .then((response) => {
-                setEmployees(response.data)
+                if(typeof(response.data) === typeof(employees)) {
```

Comment:
Nice condition ;) what if is false ? show some error message, or so ?

Reply:
Done.